firmware/common/units.hpp

```diff
@@ -24,6 +24,9 @@
 
 #include <cstdint>
 
+#define STR_DEGREES_C "\xB0\x43"  // ºC - 0xB0 is degree ° symbol in our 8x16 font, 0x43 is "C"
```

Comment:
Not pretty, but the C compiler doesn't like "\xB0C", or "\x00B0C".  The compiler version doesn't accept "\x{B0}C" (supported in C23+ only).  Clang rejects "\xB0""C".  The obvious "\xB0" + "C" works in some places, but not in OptionsField and consumes additional code space, because compiler converts it to strcat("\xB0","C").